infra/main.bicep

```diff
@@ -743,7 +743,8 @@ output AZURE_STORAGE_UPLOAD_CONTAINER string = uploadContainerName
 output BACKEND_URI string = backend.outputs.uri
 output BACKEND_NAME string = backend.outputs.name
 output RESOURCE_GROUP_NAME string = rg.name
-output AZURE_OPENAI_CHAT_GPT_DEPLOYMENT string = !empty(chatGptDeploymentName) ? chatGptDeploymentName : !empty(chatGptModelName) ? chatGptModelName : 'gpt-35-turbo-16k'
+output AZURE_OPENAI_CHAT_GPT_DEPLOYMENT string = !empty(chatGptDeploymentName) ? chatGptDeploymentName : chatGptModelName
```

Comment:
This is a regression and needs the latest updates applied, as chatGptModelName can be blank.

Reply:
Fixed. Regression didn't exist before ;)  Review has taken a while and it seems new changes occured in vNext which were pulled in later.